simson/common/data_transformations.py

```diff
@@ -7,3 +9,60 @@ def broadcast_trailing_dimensions(array: np.ndarray, to_shape_of: np.ndarray) ->
     b_reshaped = np.reshape(array, new_shape)
     b_broadcast = np.broadcast_to(b_reshaped, to_shape_of.shape)
     return b_broadcast
+
+
+class Bound:
+    """Class representing bounds for a single parameter."""
+
+    def __init__(
+        self,
+        var_name: str,
+        lower_bound: Union[float, np.ndarray],
+        upper_bound: Union[float, np.ndarray],
+    ):
+        self.var_name = var_name
+        self.lower_bound = np.array(lower_bound)
+        self.upper_bound = np.array(upper_bound)
+
+    @staticmethod
+    def create_bounds_arr(
```

Comment:
There is no reason to place this method inside the Bound class. It does not use its data, and is not called from its methods. Placing it here is therefore confusing, as one expects it to be connected to the class.